CHANGELOG.rst

```diff
@@ -37,6 +37,7 @@ Since last release
 * fix sell_policy that was offering bids when capacity was inbetween 0 and the
   quantize, bids that one was not able to fullfill and caused cyclus to crash. (#1552)
 * Deprecation warnings involving <boost/detail/sp_typeinfo.hpp> (#1611)
+* Segmentation faults when calling Cbc (#1614)
```

Comment:
```suggestion
* Resolves segmentation faults when calling Cbc (#1614)
```

Reply:
Addressed this, as well as the previous fix in `CHANGELOG.rst` that was similarly worded